app/controllers/spree/api/base_controller.rb

```diff
@@ -0,0 +1,130 @@
+require_dependency 'spree/api/controller_setup'
+
+module Spree
+  module Api
+    class BaseController < ActionController::Metal
+      include Spree::Api::ControllerSetup
```

Comment:
I know this is an exact copy of the line in the Spree codebase and I don't suggest we change it here but for everyone to be aware, it is redundant as this could be written as `include ControllerSetup`. I'd like to remove these and other redundancies once we own the code completely...

Reply:
and I hope soon this will be under only Api:: ;-)
Not long before we can kill the Spree namespace altogether in routes, controllers and views!!!

after spree_frontend is gone, the checkout (and orders, etc) controllers are ready to be moved from controller/spree to controllers/ and now the spree/api stuff is the same and can be moved from controller/spree/api to controllers/api 🎉

Reply:
thanks for leading this @luisramos0 !